src/lwf/lib/fnlwfapi.vcxproj

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectGuid>{e73b68e4-1ed8-462f-a084-b5f68bef765f}</ProjectGuid>
+    <TargetName>fnlwfapi</TargetName>
+    <UndockedType>dll</UndockedType>
+    <UndockedDir>$(SolutionDir)submodules\undocked\</UndockedDir>
+    <UndockedOut>$(SolutionDir)artifacts\</UndockedOut>
+    <UndockedUseDriverToolset>true</UndockedUseDriverToolset>
+    <UndockedSourceLink>true</UndockedSourceLink>
+  </PropertyGroup>
+  <Import Project="$(UndockedDir)vs\windows.undocked.props" />
+  <Import Project="$(SolutionDir)src\wnt.cpp.props" />
+  <ItemGroup>
+    <ClCompile Include="ioctl.c" />
+    <ClCompile Include="fnlwfapi.c" />
+    <FilesToPackage Include="$(TargetPath)" />
+  </ItemGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>
+        $(SolutionDir)inc;
+        $(SolutionDir)src\common\inc;
+        $(SolutionDir)src\lwf\inc;
+        $(IntDir);
+        %(AdditionalIncludeDirectories)
+      </AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>
+        POOL_NX_OPTIN_AUTO=1;
```

Comment:
Ideally move these POOL_* macros to a .props file - either within wnt.cpp.props or a wnt.cpp.kernel.props.